lib/loaders/total.js

```diff
@@ -19,8 +14,14 @@ export const total = (path, options = {}) => {
 
 export const totalLoader = httpLoader(total);
 
-const load = (path, options = {}) =>
-  totalLoader.load(path, options)
+const load = (path, options = {}) => {
+  const key = toKey(path, assign({}, options, {
+    size: 0,
```

Comment:
Is this change deployed to gravity production? I think that's probably the only thing holding up this PR?